starfish/image/_learn_transform/transforms_list.py

```diff
@@ -0,0 +1,103 @@
+import json
+from typing import List, Mapping, Tuple
+
+import numpy as np
+from skimage.transform._geometric import GeometricTransform, SimilarityTransform
+from slicedimage.io import resolve_path_or_url
+
+from starfish.config import StarfishConfig
+from starfish.types import Axes, TransformType
+
+
+transformsTypeMapping = {
+    TransformType.SIMILARITY: SimilarityTransform
+}
+
+
+class TransformsList:
+    """Simple list wrapper class for storing a list of transformation
+    objects to apply to an Imagestack"""
+
+    def __init__(self,
+                 transforms_list: List[Tuple[Mapping[Axes, int],
+                                             TransformType,
+                                             GeometricTransform]] = None
+                 ):
+        """
+        Parameters
+        ----------
+        transforms_list: List[Tuple[Mapping[Axes, int], SimilarityTransform]]
+            A list of tuples containing axes of an Imagestack and associated
+            transform to apply.
+        """
+
+        if transforms_list:
+            self.transforms = transforms_list
+        else:
+            self.transforms: List[Tuple[Mapping[Axes, int],
+                                        TransformType, GeometricTransform]] = list()
+
+    def append(self,
+               selectors: Mapping[Axes, int],
+               transform_type: TransformType,
+               transform_object: GeometricTransform
+               ) -> None:
+        """
+        Adds a new TransformationObject to the list
```

Comment:
`TransformationObject` is not a thing, is it?

Reply:
I just meant it to refer to the GeometricTransform class and subclasses, I'll make it more clear though